packages/config/src/memory/memory.config.ts

```diff
@@ -113,11 +114,47 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
           this.imageryToChildTileSet(obj, layer, EpsgCode.Google);
         }
       } else if (isConfigImagery(obj)) {
-        // TODO should this really overwrite existing tilesets
         this.put(ConfigProviderMemory.imageryToTileSet(obj));
-        this.imageryToTileSetByName(obj);
+        const tileSet = this.imageryToTileSetByName(obj);
+        allLayers.push(tileSet.layers[0]);
       }
     }
+    // Create an all tileset contains all raster layers
+    if (allLayers.length) this.createVirtualAllTileSet(allLayers);
+  }
+
+  /** Get the last id from the s3 path and compare to get the latest id based on the timestamp */
+  findLatestId(idA: string, idB: string): string {
+    const ulidA = ConfigId.unprefix(ConfigPrefix.Imagery, idA);
+    const ulidB = ConfigId.unprefix(ConfigPrefix.Imagery, idB);
+    try {
+      const timeA = decodeTime(ulidA);
+      const timeB = decodeTime(ulidB);
+      if (timeA >= timeB) return idA;
+      else return idB;
+    } finally {
+      //If not ulid return the idB
+      return idB;
```

Comment:
would be good to localeCompare idA vs idB so then its still alphabetical.

so then `findLatestId("a", "b")` is the same output as `findLatestId("b", "a")`

This function does not use `this` so maybe just export it as a standalone function.